gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -133,8 +134,6 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
             warnings.warn("client_cert_source is deprecated", DeprecationWarning)
 
         if channel:
-            # Ignore credentials if a channel was passed.
```

Comment:
@ohmayr I wound up removing this again, as you had originally. I convinced myself that this will not break the API surface, and we do need it for testing unless we make major changes to the architecture.

Reply:
Update: now I'm putting it back, because it looks like removing it screwed u pother tests. I think we could make a case for correct behavior either way, but keeping it in is fine. And I think I was able to finesse the changes in the new test from leaving it in.